jsapp/js/components/modalForms/submission.es6

```diff
@@ -52,14 +52,17 @@ class Submission extends React.Component {
   }
   componentDidMount() {
     this.getSubmission(this.props.asset.uid, this.state.sid);
-    this.listenTo(actions.resources.updateSubmissionValidationStatus.completed, this.refreshSubmission);
+    this.listenTo(actions.resources.updateSubmissionValidationStatus.completed, this.refreshSubmissionValidationStatus);
```

Comment:
If I understand correctly, this component has access to `this.listenTo` (which is a redux thing) because at the bottom we attach a `Reflux.ListenerMixin`

What is the reason we use mixins, and not simply have the component extend `Reflux.Component`?

Reply:
Your guess is correct :) I think mixins are "the old way" and I'm just following what's already in the code.